usaspending_api/common/api_request_utils.py

```diff
@@ -212,6 +212,11 @@ def create_q_from_filter(self, filt):
                 else:
                     # Otherwise, use built in django in
                     operation = "__in"
+            if operation is '__icontains' and isinstance(value, list):
```

Comment:
It's a bit unclear what this means from a user perspective...does that mean that the search for business types will be case sensitive?  Would putting in a test case for this type of array field searching might be a good way to make the expected behavior clearer while making sure we have coverage for this new feature?